src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -67,7 +67,11 @@ export const TransactionHistoryCardComponent: FunctionComponent<TransactionHisto
               <View style={styles.transactionText}>
                 <AppText style={styles.categoryName}>{item.name}</AppText>
                 <View style={styles.transactionText}>
-                  <AppText style={styles.quantityText}>
+                  <AppText
+                    style={{
+                      ...styles.quantityText,
+                    }}
+                  >
```

Comment:
hmm, can I check what's reason behind this change haha

Reply:
sorry was trying something and rendering. have reverted back.